consensus/istanbul/backend/handler.go

```diff
@@ -48,10 +52,14 @@ const (
 	istanbulGetAnnouncesMsg        = 0x16
 	istanbulGetAnnounceVersionsMsg = 0x17
 	istanbulAnnounceVersionsMsg    = 0x18
+	istanbulVersionedEnodeMsg      = 0x19
+	istanbulValidatorProofMsg      = 0x1a
```

Comment:
I also shared this concern, as it does feel weird. The eth handshake message is msg code `0x00` so I thought maybe of putting this as the first in the istanbul package and excluding it from the istanbul message range. What do you think of that? I was hesitant because it will require reassigning other msg codes but if we aren't worried about backward compatibility it should work

Reply:
Looking at this a bit more, I think this makes more sense to me and seems like the best solution.